core/modules/telemetry/telemetry.telemetry.inc

```diff
@@ -55,5 +60,15 @@ function telemetry_telemetry_data($key) {
       else {
         return 'Other';
       }
+    case 'modules_core':
+      $modules = array();
+      $all_modules = system_rebuild_module_data();
+      foreach (module_list() as $module) {
+        if (strpos($all_modules[$module]->uri, 'core/') === 0) {
```

Comment:
is this the best check we have to see if something is a core module? It seems as though someone could add a module in this location on their site and it would then report back as a core module.

Reply:
I vaguely recall doing something smart for a PR for either https://github.com/backdrop/backdrop-issues/issues/498 or some similar/related issue, but I cannot find anything now 😞